packages/typespec-ts/test/integration/generated/overload/src/parameters.ts

```diff
@@ -12,6 +12,7 @@ export type GetThingParameters = GetThingBodyParam & RequestParameters;
 export interface UploadBodyParam {
   /** Value may contain any sequence of octets */
   body:
+    | string
```

Comment:
This really looks unsettling.

Reply:
this is also duplicated issue and see comments https://github.com/Azure/autorest.typescript/pull/2258#discussion_r1475764471